src/nbl/video/device_capabilities/device_features.json

```diff
@@ -4138,13 +4138,20 @@
             ]
         },
         {
-            "type": "bool",
-            "name": "cooperativeMatrixRobustBufferAccess",
-            "value": false,
             "comment": [
-                "[EXPOSE AS LIMIT] redundant",
-                "bool cooperativeMatrix = limits.cooperativeMatrixSupportedStages.any();",
-                "leaving as a feature because of overhead"
+                "VK_KHR_cooperative_matrix"
+            ],
+            "entries": [
+                {
+                    "type": "bool",
+                    "name": "cooperativeMatrix",
+                    "value": false
+                },
+                {
+                    "type": "bool",
+                    "name": "cooperativeMatrixRobustBufferAccess",
+                    "value": false
```

Comment:
you want to declare the `cooperativeMatrix` feature as `MOVE_TO_LIMIT` (add the `expose` tag)

while `cooperativeMatrixRobustBufferAccess` remains a feature like it is right now